### ISSUE TITLE
Stored properties cannot be marked unavailable with '@available'

### ISSUE THREAD

**Issue by MiyoAlpizar** [PiXeL16/PasswordTextField#36]
Hi everyone, I have this version, I tried to update version but it seems this is the last one available 
pod 'PasswordTextField','1.2.0'

My project can't compile because of this error
Stored properties cannot be marked unavailable with '@available'
/**
     *  Shows the toggle button while editing, never, or always. The possible values to set are "editing", "never", "always
     */
    @available(*, unavailable, message: "This property is reserved for Interface Builder. Use 'showButtonWhile' instead.")
    @IBInspectable var showToggleButtonWhile: String? {
        willSet {
            if let newShow = ShowButtonWhile(rawValue: newValue?.lowercased() ?? "") {
                self.showButtonWhile = newShow
            }
        }
    }

**Comment by OsenLiu** [comment]
I have the same issue.

**Comment by PiXeL16** [comment]
@ricky641 Could this be related to any of your last changes?

**Comment by Abdelilahskali** [comment]
hello PiXeL16
i have the same issue with Xcode 15
Are you tested you frameWork with Xcode 15

**Comment by OsenLiu** [comment]
temporary solution: pull the latest code with following line in the podfile and run pod install.
pod 'PasswordTextField', :git => 'https://github.com/PiXeL16/PasswordTextField.git', :branch => 'master'

**Comment by ricky641b** [comment]
> @ricky641 Could this be related to any of your last changes?

Yes, I fixed this. You might need to bump the version

**Comment by ricky641b** [comment]
> hello PiXeL16 i have the same issue with Xcode 15 Are you tested you frameWork with Xcode 15

Yes, I tested with Xcode 15. You might need to remove the package and add the package again so that it can pull the latest changes.

**Comment by PiXeL16** [comment]
Thanks @ricky641b. Can someone confirm so I can close ?

**Comment by ricky641b** [comment]
> Thanks @ricky641b. Can someone confirm so I can close ?

I don't know why in the tags in the repo it is still showing 1.2.0. Can you please try to tag and update the version?

**Comment by yonat** [comment]
@PiXeL16 Would you push a new pod version please? Tx.

**Comment by PiXeL16** [comment]
1.3.0 is up now.
